website/docs/r/saml_auth_backend.html.md

```diff
@@ -0,0 +1,74 @@
+---
+layout: "vault"
+page_title: "Vault: vault_saml_auth_backend resource"
+sidebar_current: "docs-vault-saml-auth-backend"
+description: |-
+Manages SAML Auth mounts in Vault.
+---
+
+# vault\_saml\_auth\_backend
+
+Manages a SAML Auth mount in a Vault server. See the [Vault
+documentation](https://www.vaultproject.io/docs/auth/saml/) for more
```

Comment:
Thanks for putting this link to the Vault docs! I often wish all the TFVP resources did this because I usually want to cross reference.